tests/api/entities/history.test.js

```diff
@@ -1,19 +1,21 @@
 require('should')
-const { adminReq, shouldNotBeCalled } = require('../utils/utils')
+const { adminReq, dataadminReq, publicReq, authReq, shouldNotBeCalled } = require('../utils/utils')
 const { createHuman } = require('../fixtures/entities')
+const { getDeanonymizedUser, customAuthReq } = require('../utils/utils')
 const endpoint = '/api/entities?action=history'
+const { _id: anonymizedId } = require('db/couchdb/hard_coded_documents').users.anonymized
```

Comment:
this list is supposed to be actual user doc in db. which is not the same as obfuscating an id
```suggestion
const anonymizedId  = 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
```

Reply:
it's not only obfuscating, it being able to respond something when being requested on [`GET /api/users?action=by-ids`](https://inventaire.io/api/users?action=by-ids&ids=00000000000000000000000000000001), to lower the different cases to handle in clients

Reply:
why would there be any need to request such a user ? it lowers different cases maybe but if we could spare some requests, its kind of attractive no ?

Reply:
this is already expected by https://github.com/inventaire/inventaire-client/pull/275: if a non-admin user goes on the history page of an entity edited by anonymized users, patches will arrive with `user: ${anonymizedId}`, and make a `GET /api/users?action=by-ids` request for that id

Reply:
Why not deleting the `user` key in patches, and let clients fill what they cannot have access to ?

Reply:
that's a possible implementation, but we already have this existing mechanism to express special users, I find it cleaner to go with this implementation

Reply:
I tried without the special user in c8b258f1e, client update in https://github.com/inventaire/inventaire-client/pull/275/commits/ece525a2d23a1bdef07a2ec7155938da8a2d479d